### DISCUSSION TITLE
Notesnook vs Standard Notes?!

### DISCUSSION THREAD

**Discussion by Devious3901** [streetwriters/notesnook#2698] [Q&A]
Hi dear Notesnook community. I need your help. Im switched from prop soft to open source, and i need task manager tool/ todo list
![image](https://github.com/streetwriters/notesnook/assets/103393069/52878387-9cae-4e19-9ed6-7c41118d5c7b)
![image](https://github.com/streetwriters/notesnook/assets/103393069/db30e84d-029a-49c1-9550-cb3fa1f1a819)
And i dont know what is better -_-
Q1: Why u choose Notesnook?
Q2: U make test between Notesnook and Standard Notes?
Q3: What i need to choose and why?

**Comment by Bizzet** [comment]
Download both and begin using each as if it was the only option. 

Then switch to the other and do the same thing.

You will know which one feels better to you after you do.
